scalameta/contrib/shared/src/main/scala/scala/meta/contrib/implicits/DefnExtensions.scala

```diff
@@ -0,0 +1,18 @@
+package scala.meta.contrib.instances
+
+import scala.meta.Defn
+import scala.meta._
+import scala.meta.contrib.Extract
+import scala.meta.contrib.implicits.Equality._
+
+trait DefnExtensions {
```

Comment:
Why have this in a file DefnExtensions? IMO it can just sit in ExtractExtensions. Only Defn's have mods, but we do not actual require this. All we care about is that `Extract[A, Mod]` is defined.